src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1755,6 +1755,9 @@ If such a class is used as a base class and if the deriving class defines a dest
   <data name="ERR_InterfaceImplementedByConditional" xml:space="preserve">
     <value>Conditional member '{0}' cannot implement interface member '{1}' in type '{2}'</value>
   </data>
+  <data name="ERR_InterfaceImplementedImplicitlyByVariadic" xml:space="preserve">
+    <value>'{0}' cannot implement interface member '{1}' in type '{2}' because it has an __arglist parameter. Use an explicit interface implementation.</value>
```

Comment:
>Use an explicit interface implementation. [](start = 110, length = 41)

This is somewhat inaccurate, I think. Can't implicit implementation still be used in some scenarios? I mean one can provide a new implicit implementation within the current type declaration.

Reply:
This is a very extreme corner case involving undocumented feature. I thing the depth of the error message is good enough.

---
In reply to: [167041130](https://github.com/dotnet/roslyn/pull/24679#discussion_r167041130) [](ancestors = 167041130,167039605)

Reply:
Perhaps should not suggest anything at all. In some scenarios explicit implementation is not really possible. - If user actually _needs_ to bridge to the base method.

---
In reply to: [167087930](https://github.com/dotnet/roslyn/pull/24679#discussion_r167087930) [](ancestors = 167087930,167041130,167039605)

Reply:
Will remove the suggestion part.

---
In reply to: [167089732](https://github.com/dotnet/roslyn/pull/24679#discussion_r167089732) [](ancestors = 167089732,167087930,167041130,167039605)